samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -120,6 +110,7 @@ It performs the following major tasks:
 * Constructs timestamped sensor sample and location `device messages <nRF Cloud Device Messages_>`_ using `cJSON`_.
 * Sends sensor sample and location device messages to the :ref:`nrf_cloud_mqtt_multi_service_device_message_queue`.
 * Checks for and executes :ref:`remote modem AT command requests <nrf_cloud_mqtt_multi_service_remote_at>`.
+* Sends temperature and sample start alerts using the :ref:`lib_nrf_cloud_alert` library.
```

Comment:
```suggestion
* Sends the temperature and sample start alerts using the :ref:`lib_nrf_cloud_alert` library.
```
Not sure I understand this correctly. Does the thread send "sample start" alerts? Or does the sample start generating alerts?

Reply:
It sends "temperature alerts" and  "sample start alert(s)". The phrasing is admittedly hard to follow, and whether "sample start alerts" should be plural is especially unclear to me since exactly one is sent, but it is sent every time the sample boots.

Perhaps:

`Sends temperature alerts and a sample startup alerts using the...` would be clearest. I've switched to that, let me know if it is acceptable